### ISSUE TITLE
Improve adding multiple items to the database

### ISSUE THREAD

**Issue by Tecfan** [bartervg/barter.vg#98]
![image](https://user-images.githubusercontent.com/956878/76681479-7a356780-65f3-11ea-87d8-c740bf942656.png)

I (with giveaway adding rights) just tried the "add multiple games" textbox for the first time, and the process was confusing.

To be able to submit the games you need to put something in the "Game title" input field, but this input field will not be used to create a new item.

Secondly, when you successfully add a title, it is not hyperlinked. 

So two concrete suggestions for improvement:

1) If using the "add multiple games" textbox, ignore the check for the "Game title" input field if it's not even going to be used.

2) Add links to the item id's after adding games for easy verification

And further:

3) this textbox would be even more useful if it supported a csv styled input, for example:

platform,type,title,unique ID,URL ID (with support for empty fields somehow).

specifically because itch.io games are so much more useful with an URL on the game pages.

and 4) improve ability to add item connections and set free status.

**Comment by bartervg** [comment]
> this input field will not be used to create a new item.

You're right, it's effectively different 2 forms. If the multiple game textarea is used, it overrides the Game title, unique ID and URL ID. 

> it is not hyperlinked

This is setup for Steam only. However, adding this URL IDs will make it easier to add links when each platform is setup.

> csv styled input

This could lead to problems. Although uncommon, there are 1693 titles in the database with commas. This would require escaping commas or a different separator.

> itch.io games are so much more useful with an URL 

Unfortunately, because itch.io uses subdomains, I don't think the current system will work. Probably, this is its own issue.
